Fix sbindir paths in config files
                                                                                                                     
Change the default PATH of apps to reflect the location of sbindir
in /usr/sbin
                                                                                                                     
Signed-off-by: Amy Fong <amy.fong@windriver.com>
---
 audisp/plugins/remote/au-remote.conf |    2 +-
 init.d/auditd.conf                   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/audisp/plugins/remote/au-remote.conf
+++ b/audisp/plugins/remote/au-remote.conf
@@ -5,7 +5,7 @@
 
 active = no
 direction = out
-path = /sbin/audisp-remote
+path = /usr/sbin/audisp-remote
 type = always
 #args =
 format = string
--- a/init.d/auditd.conf
+++ b/init.d/auditd.conf
@@ -10,7 +10,7 @@
 freq = 20
 num_logs = 5
 disp_qos = lossy
-dispatcher = /sbin/audispd
+dispatcher = /usr/sbin/audispd
 name_format = NONE
 ##name = mydomain
 max_log_file = 6 
